list.java

```diff
@@ -0,0 +1,38 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+import org.json.simple.JSONArray; 
+import org.json.simple.JSONObject; 
+import org.json.simple.parser.*; 
+
+public class list {

```

Comment:
Conor and I briefly looked into this, but couldn't find any way to create just functions in a file with out a class.  The only thing we could find is to use "extends list" in the CMI, and it worked fine.  Do you know how to do it without the class list?  Cause it would be awesome if we didn't need the class.

Reply:
Hmm yeah I just took a look, seems it might be required to keep methods confined within classes.